packages/app/lib/src/features/home/data/post_repository.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:appwrite/appwrite.dart';
+import 'package:appwrite/models.dart';
+import 'package:flutter/material.dart';
+import 'package:latlong2/latlong.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../../../env/env.dart';
+import '../../../utils/api.dart';
+import '../domain/post_entity.dart';
+
+part 'post_repository.g.dart';
+
+/*
+How to do things:
+Put database and collection id in .env
+Implement post code from other branch
+For first implementation have post_repository read in all posts and then add queries and other filters down the line
+Have flutter access user location
+*/
+
+// Might need to have 2 posts lists. One for local and one for world
```

Comment:
Should just need one list (in the service), but multiple providers.

See: [Provider](https://riverpod.dev/docs/providers/provider) and [Combining Provider States](https://riverpod.dev/docs/concepts/combining_providers). There used to be better docs, but I can't find them.